etc/adminhtml/system.xml

```diff
@@ -693,6 +693,53 @@
                 </field>
             </group>
 
+            <group id="no_results_page_configuration" translate="label" type="text" sortOrder="60" showInDefault="0" showInWebsite="0" showInStore="1">
+                <label>No Results Found Behaviour</label>
+                 <field id="no_results_page_title" translate="label" type="text" sortOrder="0" showInDefault="0" showInWebsite="0" showInStore="1">
+                    <label>Title</label>
+                    <comment>Title for No Results Page.</comment>
+                </field>
+
+                <field id="show_no_results_products_suggestions" translate="label" type="select" sortOrder="0" showInDefault="0" showInWebsite="0" showInStore="1">
+                    <label>Display Pinned Products?</label>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                    <comment>
+                        <![CDATA[
+                            Select Yes if you want to display pinned products when the user lands on the no results found page.
+                        ]]>
+                    </comment>
+                </field>
+               
+                 <field id="no_results_products_selection" translate="label" type="multiselect" sortOrder="0" showInDefault="0" showInWebsite="0" showInStore="1">
+                    <label>Source</label>
+                    <source_model>Wizzy\Search\Model\Admin\Source\CategoriesListWithHandle</source_model>
```

Comment:
Source will be,

- Select
- Categories

Upon selecting categories we will display categoriesListHandle